### PR TITLE
Remove Null characters in font name inside the generated css file

### PR DESCRIPTION
**Pull request by shadinaif**
There is an issue in `make web` regarding the generated css file. The **font-family** value contains null characters which will make configuring the font in the MathJax environment inconvenient (sometimes).

This is to fix the generated **font-family**.
~~This will also replace the values from **AmiriWeb** to **AmiriArabicMathJax** because the actual use of the font is only MathJax~~

### CODE REVIEW

File: tools/makecss.py [hunk 1]
```diff
@@ -12,11 +12,11 @@ def genCSS(font, base):
         style = "oblique"
     weight = font["OS/2"].usWeightClass
     family = font["name"].getName(nameID=1, platformID=3, platEncID=1)
-    family = str(family).replace(" ", "")
+    family = str(family).replace(" ", "").replace("\0", "")
 
     css = """
 @font-face {
-    font-family: %(family)sWeb;
+    font-family: %(family)sArabicMathJax;
```
Comment (OmarIthawi) [inline tools/makecss.py]: Please revert the change, there's no need for making this change here.

Users (i.e. UI developers, including the font) usually do that on their own.

```suggestion
    font-family: %(family)sWeb;
```

**Review by OmarIthawi — APPROVED** [review]
Please revert the font family name change, otherwise looks good to me.

File: tools/makecss.py (hunk 1 above)
Comment (shadinaif) [reply tools/makecss.py]: Done @OmarIthawi
